ref/general/what-is-open-liberty.adoc

```diff
@@ -16,26 +16,35 @@
 * The ability to extend the Open Liberty capabilities by adding custom features that use a product extension System Programming Interface (SPI).
 
+== Zero migration
+
+With the Open Liberty zero migration policy, you focus on writing your applications, not the APIs changing under you. Zero-migration architecture means that you can use existing, unmodified configuration and application files with an updated version of Open Liberty without unwanted or unexpected change in behavior. Open Liberty offers full compatibility between product versions and pluggable features so that your existing APIs and behaviors are supported in each new product version.
+
+Open Liberty never modifies user configuration files, which are fully compatible between versions. You can use a single version of your configuration files across multiple versions. Files that you created for a previous version of Open Liberty can be used with a later version. Files that you create for later versions can be used with previous versions. As a result, if all configured features are installed, you can use a single set of configuration files across multiple versions without modifications. Any configuration settings that do not apply to a particular version of Open Liberty are ignored.
+
 == Server configuration
 
-Open Liberty follows a principle of configuration by exception, which allows for succinct config files. The runtime environment operates from a set of built-in config default settings. You specify only the configuration that overrides the default settings. For more information, see link:/docs/ref/config/#serverConfiguration.html[Server Configuration].
+Open Liberty follows a principle of configuration by exception, which allows for succinct config files. The runtime environment operates from a set of built-in config default settings. You specify config only when you want to override the default settings. For more information, see link:/docs/ref/config/#serverConfiguration.html[Server Configuration].
 
-== Feature Management
+== Feature management
 
 Features are the discrete functional units with which you control the pieces of the runtime environment that are loaded into a particular server. Features provide the programming models and services that support your applications. For a full list of available features and their configuration elements, see link:/docs/ref/feature/#featureOverview.html[Server Features].
 
-== Open Liberty Guides
+== The Open Liberty auto-tuning thread pool
+
+All the application code in Open Liberty runs in a single thread pool that is called the default executor. The size of this pool is set by an auto-tuning controller, which works well for a wide range of workloads. For most applications that run on Open Liberty, it is not necessary to tune the size of the thread pool. For more information, see link:/docs/ref/general/#Threading.html[The Open Liberty thread pool].
```

Comment:
Change,
"All the application code in Open Liberty runs in a single thread pool that is called the default executor."
to,
"The (maybe say 'entire') application code in Open Liberty runs in a single thread pool that is called the default executor."

Reply:
I think "entire" might give the impression of a all the application code for a single application, when there can be multiple applications in the server. Maybe just

"Application code in Open Liberty runs in a single...."